### DISCUSSION TITLE
Sequent 8 relay board strange behavior

### DISCUSSION THREAD

**Discussion by flyboyusa** [tagyoureit/nodejs-poolController#953] [REM]
I have Sequent 8 multi relay board V5 that is acting strange.
The relays all cycle fine, except on a few of the relays, when I cycle them from ON to OFF, all of the other relays cycle OFF then ON momentarily. This is causing my heater to cycle OFF and ON whenever my chlorine pump kicks off.

Any idea why some of the relays affect other relays?
This doesn't seem like normal behavior.

<img width="1030" alt="Screenshot 2024-05-25 at 7 24 42 AM" src="https://github.com/tagyoureit/nodejs-poolController/assets/95569334/7ea7d94b-df3e-45e6-8382-68f8a1ee9868">

**Comment by sberkovitz** [comment]
Can you download Sequents command line tool for this hat from https://github.com/SequentMicrosystems/8relind-rpi, and run `8relind <id> test`  - this will help identify if its a board issue or otherwise.

**Reply by flyboyusa** [reply]
Thanks...I just ran the test and all LED's light up in sequence

**Reply by sberkovitz** [reply]
OK that's good.  Is the board resetting? What do the logs from REM say?
